components/bt/host/bluedroid/api/include/api/esp_gap_ble_api.h

```diff
@@ -18,2686 +18,4158 @@ extern "C" {
+#define ESP_LE_AUTH_BOND                0x01        /*!< Bonding: Device is bonded with the peer, storing pairing information for future secure connections. */
+#define ESP_LE_AUTH_REQ_MITM            (1 << 2)    /*!< Man-in-the-Middle (MITM) protection required: Ensures that the communication channel is secure from eavesdropping and tampering. */
+#define ESP_LE_AUTH_REQ_BOND_MITM       (ESP_LE_AUTH_BOND | ESP_LE_AUTH_REQ_MITM)   /*!<  Bonding with MITM Protection: Devices are bonded and the connection is protected against MITM attacks.*/
+#define ESP_LE_AUTH_REQ_SC_ONLY         (1 << 3)    /*!< Secure Connections (SC) only: Uses Secure Connections for stronger security compared to Legacy Pairing. */
+#define ESP_LE_AUTH_REQ_SC_BOND         (ESP_LE_AUTH_BOND | ESP_LE_AUTH_REQ_SC_ONLY)    /*!< Bonding with Secure Connections: Devices are bonded using Secure Connections. */
+#define ESP_LE_AUTH_REQ_SC_MITM         (ESP_LE_AUTH_REQ_MITM | ESP_LE_AUTH_REQ_SC_ONLY)    /*!< Secure Connections with MITM Protection: Ensures the connection uses Secure Connections and is protected against MITM attacks. */
+#define ESP_LE_AUTH_REQ_SC_MITM_BOND    (ESP_LE_AUTH_REQ_MITM | ESP_LE_AUTH_REQ_SC_ONLY | ESP_LE_AUTH_BOND)     /*!< Secure Connections with MITM Protection and Bonding: Devices are bonded using Secure Connections and are protected against MITM attacks. */
+typedef uint8_t esp_ble_auth_req_t;    /*!< Type definition for Bluetooth Low Energy authentication requirements. */
+
+/**
+ * @brief  These macros are used in settings for authentication enable or disable.
+ */
+#define ESP_BLE_ONLY_ACCEPT_SPECIFIED_AUTH_DISABLE    0   /*!< Authentication disabled. */
+#define ESP_BLE_ONLY_ACCEPT_SPECIFIED_AUTH_ENABLE     1   /*!< Authentication enabled. */
+
+/**
+ * @brief  These macros are used in settings for  Out of Band data authentication enable or disable.
+ */
+#define ESP_BLE_OOB_DISABLE    0  /*!< Out of Band data authentication disabled. */
+#define ESP_BLE_OOB_ENABLE     1   /*!< Out of Band data authentication enabled. */
+
+/**
+ * @brief These macros define the various input/output capabilities.
+ * Relate to BTM_IO_CAP_xxx in stack/btm_api.h (Reserved for internal use).
+ */
+#define ESP_IO_CAP_OUT                      0   /*!<  Display Only: Device has output capabilities only, typically a display. */
+#define ESP_IO_CAP_IO                       1   /*!< Display and Yes/No Input: Device has both output capabilities (display) and simple input capabilities (Yes/No buttons). */
+#define ESP_IO_CAP_IN                       2   /*!< Keyboard Only: Device has input capabilities only, typically a keyboard. */
+#define ESP_IO_CAP_NONE                     3   /*!< No Input and No Output: Device has neither input nor output capabilities. */
+#define ESP_IO_CAP_KBDISP                   4   /*!< Keyboard and Display: Device has both input capabilities (keyboard) and output capabilities (display). */
 
+/**
+ * @brief  These macros define the appearance.
```

Comment:
Change this to below:
`// These macros define the appearance. Relates to stack/btm_ble_api.h`